### PR TITLE
Alert&button

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review and provide your valuable feedback.

### CODE REVIEW

File: Alert/Alert.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  
+  .fa {
+    padding-right: 1rem;
+  }
+  
+  .closebtn {
+    color: white;
+    cursor: pointer;
+    float: right;
+    font-size: 22px;
```
Comment (muskanbagrecha) [inline Alert/Alert.css]: Give rem for font size, avoid using float if possible. Alternative would be to use flex.

**Comment by muskanbagrecha** [comment]
Can also increase the font size of alerts just a little bit to make it legible.
Overall size can also be increased to give a neat look. 
The colors of alerts can be toned down slightly. Check contrast.

**Comment by rahulyadav139** [comment]
in button with text and icon, the icon is not vertically center. You can use vertical-align: middle. Use rem instead of px.

File: Alert/Alert.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    margin-left: 15px;
+    padding-top: 0.8rem;
+    transition: 0.3s;
+  }
+  
+  .closebtnjs {
+    color: white;
+    cursor: pointer;
+    float: right;
+    font-size: 22px;
```
Comment (saurabhchirde) [inline Alert/Alert.css]: anywhere using size above 10px, use rem.

File: Alert/Alert.css (hunk 1 above)
Comment (ShrutiLakhani) [reply Alert/Alert.css]: Made the changes as suggested

File: Alert/Alert.css (hunk 2 above)
Comment (ShrutiLakhani) [reply Alert/Alert.css]: Made the changes.

**Review by ShrutiLakhani — COMMENTED** [review]
Ignore

**Comment by rahulyadav139** [comment]
After making changes, looks good to me. Go for merge.

**Comment by saurabhchirde** [comment]
Now it's looking good to me. You can merge it.

File: Alert/Button/Button.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        <button class="primary-button-outline">PRIMARY-OUTLINE</button>
+        <button class="primary-button-link">PRIMARY-TEXT</button>
+      </div>
+       <!-- Button with Icons-->
+      <div class="button-container">
+        <button class="primary-button-icon">
+          <span class="material-icons"> shopping_bag</span>
+        </button>
+        <button class="primary-button-icon-text">
+          <span class="material-icons"> shopping_bag</span>ADD TO BAG
```
Comment (gadgetfather) [inline Alert/Button/Button.html:29]: You can remove spaces

**Review by gadgetfather — CHANGES_REQUESTED** [review]

**Review by rahulyadav139 — APPROVED** [review]
Everything is great. Go ahead to merge the branch.

**Review by gadgetfather — APPROVED** [review]

**Pull request merged**